### ISSUE TITLE
Component Request : Behavioral Sources and Labels

### ISSUE THREAD

**Issue by Awen2345** [Proto-App/Proto-Android#192]
Please add Component
Behavioral Source
Labels

**Comment by Proto-App** [comment]
@Awen2345 
Do you mean behavioral source like in LTSpice?
We have labels, you can use "Text" component.

**Comment by Awen2345** [comment]
@Proto-App yes!!!! Because the source behavior is used when creating the subcircuit

**Comment by Awen2345** [comment]
@Proto-App and Internal Node its used When for source behavior

**Comment by Awen2345** [comment]
@Proto-App
